spec/spec_helper.rb

```diff
@@ -30,6 +30,12 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.around(:each, :run_delayed_jobs) do |example|
```

Comment:
added this nice helper. it runs the jobs synchronously so you get error reporting inline with the test/logging output